src/base/utils/net.h

```diff
@@ -28,13 +28,23 @@
 
 #ifndef BASE_UTILS_NET_H
 #define BASE_UTILS_NET_H
+
 class QString;
 
 namespace Utils
 {
     namespace Net
     {
         bool isValidIP(const QString &ip);
+
+        template <typename T>
+        T unquote(const T &str)
```

Comment:
Why can't we implement common function? E.g. (assuming template is valid only for QString and QStringRef as in your code):
```c++
template <typename T>
T unquoted(const T &str, const QString &quotes = QLatin1String('"'))
{
    if (str.length() < 2) return str;

    for (auto const quote : quotes) {
        if (str.startsWith(quote) && str.endsWith(quote))
            return str.mid(1, str.length() - 2);
    }
    
    return str;
}
```